datalab/datalab_session/data_operations/median.py

```diff
@@ -54,9 +54,9 @@ def operate(self):
             self.set_operation_progress(0.5 * (index / len(input_list)))
 
         # Creating the Median array
-        cropped_data = crop_arrays([image.sci_data for image in input_fits_list])
-        stacked_ndarray = np.stack(cropped_data, axis=2)
```

Comment:
np.stack doubles memory usage of the image data, and can be avoided by taking a median of flattened arrays and then reshaping back to the original cropped size.